docs/src/pages/zh-CN/getting-started.md

```diff
@@ -50,14 +61,24 @@ Astro 是一个现代的静态网站生成工具. 你可以从 [我们的主页]
 
 [@astrodotbuild](https://twitter.com/astrodotbuild) Twitter 账户是 Astro 团队的官方更新来源。
 
+<<<<<<< HEAD
```

Comment:
I looks like there are quite a few merge conflicts in these changes, can you resolve these please?

Reply:
Of course, sorry for not noticing a conflict with the main branch